index.html

```diff
@@ -0,0 +1,321 @@
+          </div>
+          <div class="p-text">
+            <p class="word">
+              Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+            </p>
+          </div>
+          <div>
+            <ul class="languages">
+              <li>html</li>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <img src="./images/Hover.png" alt="button" class="see">
+        </div>
+      </div>
+      <div class="card2">
+        <div class="snapt">
+          <img src="./images/snapshot2.png" alt="snapshot" class="responsive">
+        </div>
+        <div class="titol">
+          <div class="w-title">
+            <h2 class="name">Uber Navigation</h2>
+            <ul class="canop">
+              <li>Uber</li>
+              <li>Lead developer</li>
+              <li>2018</li>
+            </ul>
+          </div>
+          <div class="p-text">
+            <p class="word">
+              A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
+            </p>
+          </div>
+          <div>
+            <ul class="languages-2">
+              <li>html</li>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <img src="./images/see-project.png" alt="button" class="blind">
+        </div>
+      </div>
+    </section>
+    <section class="about-me">
+      <div class="me">
+        <h2 class="myself">About Myself</h2>
+        <p class="hello">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <div>
+          <h4 class="connect">LET'S CONNECT</h4>
+          <ul class="icon">
+            <li>
+              <a href="#"
+                ><img src="./images/twitter.png" alt="twitter-icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/linkedin.png" alt="linkedin-icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/medium.png" alt="medium-icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/github.png" alt="github-icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/angellist.png" alt="angellist-icon"
+              /></a>
+            </li>
+          </ul>
+          <img src="./images/resume.png" alt="button" class="press">
+        </div>
+        <div class="lang">
```

Comment:
- [ ] You have used an image in place of a `<button>` tag for the "Get my resume" button. Please remove the image and use a button tag instead. Do this for all your images.
- [ ] Also kindly implement the button default, hover and active states as shown in the Figma design below 👇 
![image](https://user-images.githubusercontent.com/46839548/166105675-ca7e6c0d-54bf-4cb3-ad34-5f073ab046d1.png)